tests/compact-manifest.jsonld

```diff
@@ -869,6 +869,33 @@
       "context": "compact/0103-context.jsonld",
       "expect": "compact/0103-out.jsonld",
       "option": {"processingMode": "json-ld-1.1", "specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#t0104",
+      "@type": ["jld:PositiveEvaluationTest", "jld:CompactTest"],
+      "name": "Compact @type with @container: @set",
+      "purpose": "Ensures that a single @type value is represented as an array",
+      "input": "compact/0104-in.jsonld",
+      "context": "compact/0104-context.jsonld",
+      "expect": "compact/0104-out.jsonld",
+      "option": {"processingMode": "json-ld-1.1", "specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#t0105",
+      "@type": ["jld:PositiveEvaluationTest", "jld:CompactTest"],
+      "name": "Compact @type with @container: @set using an alias of @type",
+      "purpose": "Ensures that a single @type value is represented as an array",
+      "input": "compact/0105-in.jsonld",
+      "context": "compact/0105-context.jsonld",
+      "expect": "compact/0105-out.jsonld",
+      "option": {"processingMode": "json-ld-1.1", "specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#t0106",
```

Comment:
Won't this test be a negative test in 1.0 mode?

Reply:
I think the tests are reasonable to check both 1.0 and 1.1 modes. The negative test is in expansion, to make sure a 1.0 processor fails. A 1.0 processor compacting with an alias of `@type` will allow a `@container` definition but ignore it when actually compacting.

Reply:
Ah, I thought it failed in 1.0.

Reply:
We actually never checked that keyword aliases should only be for simple term definitions; we just ignore everything else. We could consider tightening these up in 1.1, and require that keyword aliases only be simple term definitions.